test/dump/relocations-long-func-bodies.txt

```diff
@@ -0,0 +1,98 @@
+;;; TOOL: run-objdump
```

Comment:
Is there something about long function bodies that was producing incorrect output before?